### PR TITLE
Ietad

### PR DESCRIPTION
**Pull request by abelzer212**
IET Measure

### CODE REVIEW

File: services/ui-src/src/measures/2021/IETAD/validation/types.ts [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+    GeographyOptions: string[];
+    UrbanGeographyRate: AggregateRate;
+    RuralGeographyRate: AggregateRate;
+    AddtnlGeographyDesc: string;
+    AddtnlGeographyRate: AggregateRate;
+
+    ACAGroupRate: AggregateRate;
+  }
+}
```
Comment (benjaminpaige) [inline services/ui-src/src/measures/2021/IETAD/validation/types.ts:311]: Can double check to make sure that all of the types in this file are used within the measure? Obviously we want to make sure these are all represented in the measure when we hand this to mathmatica

Comment (abelzer212) [reply services/ui-src/src/measures/2021/IETAD/validation/types.ts:311]: Yes this is part of the OPM

**Review by jdinh8124 — APPROVED** [review]
Looks good and the global validations have already helped me in tons of other measures

File: services/ui-src/src/measures/2021/IETAD/validation/types.ts [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    "DataSource-Administrative-Other"?: string;
+    "DataSource-Administrative-Other-Explanation"?: string;
+    "DataSource-Electronic": string;
+    "DataSource-Electronic-Explanation": string;
+    "DataSource-Other": string;
+    "DataSource-Other-Explanation": string;
+    "DataSource-Hybrid"?: string[];
+    "DataSource-Hybrid-Other"?: string;
+    "DataSource-Hybrid-Other-Explanation"?: string;
+    "DataSource-Hybrid-MedicalRecord-DataSoruce"?: string;
```
Comment (jdinh8124) [inline services/ui-src/src/measures/2021/IETAD/validation/types.ts:47]: I think some of the hybrid data sources could be removed

**Review by jdinh8124 — DISMISSED** [review]

**Pull request merged**